index-fetch.js

```diff
@@ -26,6 +26,9 @@ module.exports.createFastMessageEvent = createFastMessageEvent
 
 module.exports.EventSource = require('./lib/web/eventsource/eventsource').EventSource
 
+const api = require('./lib/api')
```

Comment:
From my local testing, it seems only api-connect is missing. But my testing isn't very thorough, so I am not entirely sure if other methods may also be needed in other paths, in any case it seems a bit of a footgun to leave the Dispatcher partially complete in the Node.js bundle, so I included them all. I am not sure if it's against any principle to include the `lib/api` parts in the Node.js, bundle though. Locally it increases the bundle size from 610KB+ to 650KB+

Reply:
Also for context, this is done for the public bundle https://github.com/nodejs/undici/blob/ff18d8c9b213e1ea95f7e9917afcf8fc056b8be7/index.js#L25 (and funny enough if seems corepack once used internal paths instead of the bundle, so there were issues like https://github.com/nodejs/corepack/issues/417